### ISSUE TITLE
Gradle projected generated by the Open Liberty starter is not detected in the dev mode dashboard

### ISSUE THREAD

**Issue by kathrynkodama** [OpenLiberty/liberty-tools-vscode#111]
Generating a new Liberty Gradle project with https://start.openliberty.io/ and the Liberty dev dashboard does not detect this project

See https://github.com/OpenLiberty/open-liberty-tools-intellij/issues/63

**Comment by Jonathan-Maciel** [comment]
For future reference on a more complete solution, this is the logic I used to search the GradleBuildFile object for the string "io.openliberty.tools":
```
export function validGradleBuild(buildFile: any): GradleBuildFile {
    var returnVal = (new GradleBuildFile(false, ""));
    function searchKeys(file: any){
        for (let key in file){
            if (typeof (file[key]) == "string") {
                if (file[key].includes("io.openliberty.tools")) {
                    returnVal = (new GradleBuildFile(true, LIBERTY_GRADLE_PROJECT_CONTAINER));
                }
            } else {
                searchKeys(file[key]); 
            }
        }
    }
    searchKeys(buildFile);
    return returnVal;
}
```

**Comment by kathrynkodama** [comment]
May be fixed by #119
